src/domain/event/partial/InfoItem.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent } from 'react';
+
+import styles from './occurrenceInfo.module.scss';
+import Icon from '../../../common/components/icon/Icon';
+
+type Props = {
+  className?: string;
+  icon: string;
+  iconAlt?: string;
+  label?: string | undefined;
+};
+
+const InfoItem: FunctionComponent<Props> = ({
+  className,
+  icon,
```

Comment:
Could this be named iconPath or iconSrc so that it would be clearer what type of data is needed.

Reply:
Good idea. We're using it like this in other parts of the project, but I'll replace it here for now.